config/co2eq_parameters.json

```diff
@@ -1212,6 +1212,12 @@
       "fossilFuelRatio": 0.35349837676303403,
       "renewableRatio": 0.646501623236966
     },
+    "NKR": {
+      "_comment": "Average carbon intensity using 2016 mean production per fuel source (100% hydro) and IPCC 2014 default lifecycle emission factor",
+      "carbonIntensity": 24,
+      "fossilFuelRatio": 0.0,
+      "renewableRatio": 0.00
```

Comment:
if it's 100% hydro, doesn't it mean renewableRatio should be 1.0?

Reply:
Definitely, I patched it.